vendor.conf

```diff
@@ -1,6 +1,6 @@
 # cri dependencies
 github.com/docker/distribution                      0d3efadf0154c2b8a4e7b6621fff9809655cc580
-github.com/docker/docker                            d1d5f6476656c6aad457e2a91d3436e66b6f2251
+github.com/docker/docker                            standalone_symlink https://github.com/thaJeztah/docker.git
```

Comment:
temporarily using my fork until https://github.com/moby/moby/pull/40648 is merged